app/components/blacklight/document/bookmark_component.rb

```diff
@@ -21,6 +21,14 @@ def bookmarked?
         helpers.bookmarked? @document
       end
 
+      def bookmark_icon
+        icon = helpers.blacklight_icon("bookmark")
+        # Check if there is a bookmark icon since it is optional
```

Comment:
Can you expand on this?  How is it optional?  Where is the option?

Reply:
You have to have a bookmark.svg file in the assets/images/blacklight folder, which isn't the default. Maybe enabled or existing is better language?

Reply:
I think saying that in the comments is helpful here. Otherwise there's no way for a user to find out about this feature.